conans/client/tools/win.py

```diff
@@ -457,9 +457,9 @@ def relevant_path(path):
             keywords = "msbuild", "visual", "microsoft", "/msvc/", "/vc/", "system32", "windows"
             return any(word in path for word in keywords)
 
-        path = new_env.get("PATH", "").split(";")
-        path = [entry for entry in path if relevant_path(entry)]
-        new_env["PATH"] = ";".join(path)
+        path_key = ([name for name in new_env.keys() if "path" == name.lower()] or ["PATH"])[0]
```

Comment:
This will add ``"PATH"`` variable (empty) if it was not already there. Remove the `or ...` and check for `path_key` in order to add it to `new_env` or not in the following lines.

Reply:
Yes, but that was the previous behavior too. If we think that it shouldn't be added, that might be a breaking feature and a new issue for Conan 2.0 should be filed.